modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -165,6 +165,18 @@ public void testPostExportTask() throws Exception {
 		}
 	}
 
+	private String _splitClassName(String className) {
+		if (className.contains("#")) {
+			String[] classNameTaskItemDelegateName = className.split("#");
+
+			_taskItemDelegateName = classNameTaskItemDelegateName[1];
```

Comment:
I wouldn't say it's the best practice that a private method __splitClassName_  also sets an instance field. 

If this was a solution to return 2 values from a single method, I would suggest to use [KeyValuePair](https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/KeyValuePair.java). If that is not available on the classpath simply use some Java collection like Map or even array.

Reply:
thanks for the comment @matijapetanjek Do you mean to make it something like this?
	```
private Map<String, String> _splitClassName(String className) {
		Map<String, String> result = new HashMap<>();

		if (className.contains("#")) {
			String[] classNameTaskItemDelegateName = className.split("#");

			result.put("className", classNameTaskItemDelegateName[0]);
			result.put("taskItemDelegateName", classNameTaskItemDelegateName[1]);
		} else {
			result.put("className", className);
		}
```

Reply:
LGTM :D